pom.xml

```diff
@@ -45,10 +45,11 @@
     <dagger.version>0.9</dagger.version>
     <otto.version>1.3.1</otto.version>
     <gson.version>2.2</gson.version>
-    <junit.version>4.10</junit.version>
+    <junit.version>4.11</junit.version>
```

Comment:
Might be worth documenting this requirement on burst - I kept running into the error
`java.lang.NoSuchMethodError: org.junit.runner.Description.createTestDescription(Ljava/lang/String;Ljava/lang/String;[Ljava/lang/annotation/Annotation;)Lorg/junit/runner/Description;`